seesaw/script/run_pipeline.py

```diff
@@ -164,7 +164,12 @@ def check_and_update():
 
 
 def main():
-    parser = ArgumentParser(description="Run the pipeline")
+    global runner
```

Comment:
Or does this (like `global warrior` below) expose the variable for use outside `main`? (That seems to be the intent.)